config/runtime.exs

```diff
@@ -100,25 +100,26 @@ base_oban_cron = [
    args: %{"type" => "monthly_project_digest"}}
 ]
 
-conditional_cron =
+purge_cron =
   if System.get_env("PURGE_DELETED_AFTER_DAYS") != 0,
```

Comment:
From some previous issue, the result of `System.get_env` should be compared to a String (`"0"`).